ci/docker/common-gh200.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+packages:
+  all:
+    target: [neoverse_v2]
+  # Set nvpl as default blas, lapack and scalapack provider.
+  # Can be overwritten in environments if needed.
+  blas:
+    require: 'nvpl-blas'
+  lapack:
+    require: 'nvpl-lapack'
+  scalapack:
+    require: 'netlib-scalapack'
+  mpi:
+    ### Workaround until CE provides full MPI substitution.
+    require: 'cray-mpich'
+  pika:
+    require:
+      - "@git.6cfeeb1ba7541790134ec33a16d2b769a817b090=0.30.1"
```

Comment:
Just commenting to make sure we don't leave this in here. The fix in this commit is going to be part of pika 0.32.0 which I'll try to release today. The fix seems to help a bit with slow tests, but it doesn't seem to be the main reason why C API tests run slowly.